blueprints/ephemeral-disks/main.tf

```diff
@@ -1,5 +1,5 @@
 provider "aws" {
-  region = local.region
+  region = var.aws_region
```

Comment:
since this is a variable - we should use it directly and avoid the wrapping in a local